test/run-shared-integration-tests

```diff
@@ -6,7 +6,7 @@ set -eo pipefail
 
 declare project_root="${0%/*}/.."
 declare ringpop_common_dir="${0%/*}/ringpop-common"
-declare ringpop_common_branch="master"
+declare ringpop_common_branch="fix/test-state-transistions"
```

Comment:
This will be changed back to `master` after uber/ringpop-common#95 is merged into master